### ISSUE TITLE
FAKIcon not loading for iphone 5

### ISSUE THREAD

**Issue by RSuh** [PrideChung/FontAwesomeKit#59]
App freezes at CGFontRef newFont = CGFontCreateWithDataProvider(fontDataProvider);

**Comment by PrideChung** [comment]
Did you remove the sim card of your iPhone?

**Comment by RSuh** [comment]
Yes, could that be the issue? Does the iphone need a network connection?

**Comment by RSuh** [comment]
Hello?

**Comment by RSuh** [comment]
Under what circumstances would the app not get past that line of code?

**Comment by PrideChung** [comment]
It could be a bug of CoreText, check this issue: https://github.com/PrideChung/FontAwesomeKit/issues/25

**Comment by RSuh** [comment]
http://stackoverflow.com/questions/24900979/cgfontcreatewithdataprovider-hangs-in-airplane-mode/24901580#24901580 

Check this out! Maybe leave it in for future users who run into the bug as well!
